### ISSUE TITLE
Improve title matching

### ISSUE THREAD

**Issue by Revadike** [bartervg/barter.vg#282]
# What problem does this feature address?
I'm forced to repeat matching the same bad game titles

## Describe a solution
When you `Link Matched Titles to Profiles` save all entries in database. When there are enough submissions of the same title for the same item, add it as alternate item title, so it will get auto-matched in the future.

**Comment by Tecfan** [comment]
See also #60  and #176

**Comment by Revadike** [comment]
Yes that's also important to have, but not the same thing though.

**Comment by bartervg** [comment]
Could you provide examples?

**Comment by Revadike** [comment]
> Could you provide examples?

Sure, if only I can remember this 1 year later... 🤪

I think the main issue is, what @Tecfan already mentioned and also my described solution.

**Comment by bartervg** [comment]
This is due to clearing all tradables, then adding them back by title, and some of the titles don't match. Previously you matched those titles to Barter items, but this works is gone each time you clear the list?
